api/v1alpha1/gitrepository_types.go

```diff
@@ -41,6 +41,10 @@ type GitRepositorySpec struct {
 	// The git reference to checkout and monitor for changes, defaults to master branch.
 	// +optional
 	Reference *GitRepositoryRef `json:"ref,omitempty"`
+
+	// Verify PGP signature for the commit that HEAD points to.
```

Comment:
```suggestion
	// Verify OpenPGP signature for the commit that HEAD points to.
```

To not confuse the spec with the proprietary (Symantec's) PGP.